tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -202,6 +202,8 @@ internal TracerSettings(IConfigurationSource? source, IConfigurationTelemetry te
 
             CustomSamplingRulesInternal = config.WithKeys(ConfigurationKeys.CustomSamplingRules).AsString();
 
+            CustomSamplingRulesFormat = config.WithKeys(ConfigurationKeys.CustomSamplingRulesFormat).AsString(defaultValue: Sampling.SamplingRulesFormat.Regex);
```

Comment:
Related to my other validation comment, perhaps we can add a converter that's similar to the MetadataSchemaVersion above, so that we can assert that the configured value was `regex` or `glob` and if not then we record a telemetry failure and use the default value.

Reply:
> then we record a telemetry failure and use the default value.

We discussed offline, but commenting here for future reference. I would like to record the telemetry failure but I _don't_ want to use the default value (`regex`). Instead I wan't to disable the sampling rules.